html5/connect.html

```diff
@@ -691,7 +691,7 @@ <h4 class="panel-title">Advanced options</h4>
 	}
 
 	function json_action(uri, success_fn, error_fn) {
-		const url = get_host_address()+uri;
+		const url = `${get_host_address()}/${uri}`;
```

Comment:
How about ensuring that `get_host_address` always returns a path terminated with a `/` instead since it already does something related there:
https://github.com/Xpra-org/xpra-html5/blob/2a88f8ba1a8b8a1f95d990096f5b470395837304/html5/connect.html#L684-L690
Something like:
```
if (!path.endsWith("/")) {
    url += "/";
}
```